content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -196,47 +197,57 @@ Take a look at the script. In the simplest scenario, all we need to do on a new
-want your training code to leverage GPUs, the script will look a bit more
-complex.
+want your training code to leverage GPUs, the script will look a bit different.
 
 ## Training with GPU
 
-Setting up the environment to leverage GPU devices is a bit involved as you need
-to make sure you have the required drivers, NVIDIA-docker, and CUDA.
+There are several ways you can leverage GPU devices on a remote machine. You can install all 
+the required drivers and dependencies "manually" via a script, you can use an existing Docker image 
+or build your own, or, in the case of AWS, you can take advantage of an existing pre-configured Deep Learning AMI. 
+In this tutorial, we'll use an AMI, as it is the quickest way to start using GPUs on an AWS EC2 machine.
 
 ```hcl
 resource "iterative_task" "tpi-examples-gpu" {
-    name      = "tpi-examples-gpu"
+    name      = "ami-gpu-example"
     cloud     = "aws"
     region    = "us-east-2"
     machine   = "l+k80"
+    disk_size = "130"
+    image     = "ubuntu@898082745236:x86_64:Deep Learning AMI (Ubuntu 18.04) Version 54.0"
     directory = "."
 
     script = <<-END
     #!/bin/bash
-    sudo apt update
-    sudo apt-get install -y ubuntu-drivers-common
-    sudo ubuntu-drivers autoinstall
-
-    sudo curl -fsSL https://get.docker.com -o get-docker.sh && sudo sh get-docker.sh &&
-    sudo usermod -aG docker ubuntu
-    sudo setfacl --modify user:ubuntu:rw /var/run/docker.sock
-    curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
-    curl -s -L https://nvidia.github.io/nvidia-docker/ubuntu20.04/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
-    sudo apt update && sudo apt install -y nvidia-docker2
-    sudo systemctl restart docker
-    rm get-docker.sh
-
-    nvidia-smi
-    docker run --rm --gpus all -v "$PWD:/tpi" iterativeai/cml:0-dvc2-base1-gpu \
-        /bin/bash -c "cd /tpi; pip install -r requirements.txt; python src/train.py"
+    pip3 install -r requirements.txt
+    python3 src/train.py 
     END
 }
 ```
 
-Here, to run my training code I am passing my project's directory as a volume to
-a Docker image provided by Iterative (`iterativeai/cml:0-dvc2-base1-gpu`) which
-takes care of CUDA.
+To use a Deep Learning AMI, you need to configure it via the `image` argument that has 
+the following structure: `{user}@{owner}:{architecture}:{name}`.
+You can learn more about AMIs in the official 
+[AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
+
+---
+**NOTE**
+
+One way to find a suitable AMI, is by searching in AWS Console: 
+![AMI Catalog](/uploads/images/2022-01-15/AMI_catalog.png)
+
+Alternatively, you can use search for AMIs using [`aws ec2 describe-images` CLI command](https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/usingsharedamis-finding.html#usingsharedamis-finding-cli) with some filters, 
+e.g.: 
+```
+aws ec2 describe-images --image-ids=ami-0476bba883df7cca6 --region=us-east-2
+```
+---
+
```

Comment:
we could delete this bit to make it shorter in light of the previous comment linking to https://docs.aws.amazon.com/dlami/latest/devguide/options.html? Otherwise sounds a bit like the post is mostly talking about AWS AMIs.

```suggestion
```

Reply:
Fixed in f866dac126e9303660ce02c1934a4bc215360d49